Makefile

```diff
@@ -43,8 +46,8 @@ copy: $(RUNTIME) $(DISK_IMAGE)
 	rm -rf $(MOUNT_DIR)
 
 $(RUNTIME): $(ASM_OBJS) $(OBJS) $(SDK_EDGE_LIB) $(TMPLIB)
-	$(LINK) $(LINKFLAGS) -o $@ $^ -T runtime.lds
-	$(OBJCOPY) --add-section .options_log=.options_log --set-section-flags .options_log=noload,readonly $(RUNTIME)
+	$(LINK) $(LINKFLAGS) -o $@ $^ -T runtime.lds $(RISCV)/lib/gcc/riscv$(BITS)-unknown-elf/8.3.0/libgcc.a
```

Comment:
Could you take a quick look to see if we can remove this?
I believe the linker should choose a correct libgcc by itself.
You can investigate why the linker couldn't. Maybe post a github issue if you're unsure?

Reply:
See discussion on why we needed it on David's comment above. I changed the code to dynamically find libgcc.